docs/prototype.md

```diff
@@ -6,7 +6,7 @@ It is *not* fully built out and will likely change dramatically as the actual sp
 ```text
 Express : Activity (actor is conveying something to an audience)
     Object -> Object | Link (what is being expressed)
-    Target -> Object | Link (who is it expressed to)
+    Target -> Object | Link (what this expression is about)
```

Comment:
I think it's more semantically consistent to use the addressing fields (To/Cc/etc) to indicate who an expression is directed toward, if it's directed. That frees up Target to indicate what the expression is about. For example:

```yml
Type: Express
Object: [Happy, Excited, Relieved]
Target: 
  Type: Note
  Content: I got a job offer!
To: [actor:catboy, actor:puppygirl]
Audience: as:public
```

Reply:
@jenniferplusplus I can see the logic there, but I was basing this on the descriptions of `object` and `target` properties:

Object:
> When used within an [Activity](https://www.w3.org/TR/activitystreams-vocabulary/#dfn-activity), describes the direct object of the activity. For instance, in the activity "John added a movie to his wishlist", the object of the activity is the movie added.

Target:
> Describes the indirect object, or target, of the activity. The precise meaning of the target is largely dependent on the type of action being described but will often be the object of the English preposition "to". For instance, in the activity "John added a movie to his wishlist", the target of the activity is John's wishlist. An activity can have more than one target. 

Do you still think it makes more sense to use "To" instead of "Target"? The intended use case is to express *something* to/at *someone*, possibly with a wider audience (such as public or followers).

Reply:
That's a good point. I guess what I'm describing is more the direct object. In which case, `Object` seems like the correct field. But that's serving as a kind of subtype for Express.

Maybe then we can indicate what the expression is regarding using the `InReplyTo` field?

Reply:
I think that about/regarding association is particularly important when you get into expressing identities. That's what enables an actor to speak as someone.

Reply:
@warriordog just making sure you get the notification. I can never tell if github will send one otherwise

Reply:
@jenniferplusplus oh I just saw this, sorry! I'm planning to return to this project soon, and I'll take another look at this PR.

Reply:
@warriordog no worries! It's going to be A While :tm: before I can actually implement any of this. At this stage it's all theory crafting for me, and it's not even slightly a problem if that moves slowly.